test/DynamicHelpTest.cs

```diff
@@ -97,6 +97,13 @@ private static PSObject GetParameterHelpObject(string description)
             return paramHelp;
         }
 
+        [SkippableFact]
+        public void DynHelp_GetFullHelp_OnEmptyLine()
+        {
+            TestSetup(KeyMode.Cmd);
+            Test("", Keys(_.F1, _.Enter));
```

Comment:
Should we verify that the console in unchanged?

Reply:
I think this should be fine.